src/data/device_automation.ts

```diff
@@ -6,13 +6,15 @@ export interface DeviceAutomation {
   alias?: string;
   device_id: string;
   domain: string;
-  entity_id: string;
+  entity_id?: string;
```

Comment:
Doesn't every device automation have an entity id?

Reply:
Event based device triggers don't. Noticed it because type checking this trace failed.